tests/src/test/scala/scalaclean/deadcode/RuleSuite.scala

```diff
@@ -43,7 +43,7 @@ class RuleSuite extends SemanticRuleSuite() {
       var messages: List[RuleDiagnostic] = Nil
       rule.rules.foreach { r =>
```

Comment:
it could be possible to just call .semanticRules or .syntacticRules instead here